pkg/cmd/issue/create/create.go

```diff
@@ -0,0 +1,228 @@
+package create
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/cli/cli/api"
+	"github.com/cli/cli/git"
+	"github.com/cli/cli/internal/config"
+	"github.com/cli/cli/internal/ghrepo"
+	prShared "github.com/cli/cli/pkg/cmd/pr/shared"
+	"github.com/cli/cli/pkg/cmdutil"
+	"github.com/cli/cli/pkg/githubtemplate"
+	"github.com/cli/cli/pkg/iostreams"
+	"github.com/cli/cli/utils"
+	"github.com/spf13/cobra"
+)
+
+type CreateOptions struct {
+	HttpClient func() (*http.Client, error)
+	Config     func() (config.Config, error)
+	IO         *iostreams.IOStreams
+	BaseRepo   func() (ghrepo.Interface, error)
+
+	RepoOverride string
+	WebMode      bool
+
+	Title         string
+	TitleProvided bool
+	Body          string
+	BodyProvided  bool
+
+	Assignees []string
+	Labels    []string
+	Projects  []string
+	Milestone string
+}
+
+func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Command {
+	opts := &CreateOptions{
+		IO:         f.IOStreams,
+		HttpClient: f.HttpClient,
+		Config:     f.Config,
+	}
+
+	cmd := &cobra.Command{
+		Use:   "create",
+		Short: "Create a new issue",
+		Example: heredoc.Doc(`
+			$ gh issue create --title "I found a bug" --body "Nothing works"
+			$ gh issue create --label "bug,help wanted"
+			$ gh issue create --label bug --label "help wanted"
+			$ gh issue create --assignee monalisa,hubot
+			$ gh issue create --project "Roadmap"
+		`),
+		Args: cmdutil.NoArgsQuoteReminder,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			// support `-R, --repo` override
```

Comment:
same question about follow up work as other PR

Reply:
This already works; no follow-up needed: https://github.com/cli/cli/pull/1485/files#r467236160